pyproject.toml

```diff
@@ -40,7 +40,7 @@ ignore_missing_imports = true
 
 [tool.poetry]
 name = "spark-client"
```

Comment:
**issue** We should really change the name of the package!

Reply:
Sure. Do we want it to be called as the repository itself is named? (I'm not sure, I thought this naming was good to keep.)

Reply:
If I gather it correctly, we don't just want to change this name -- but we want to rename the library itself. (As this naming here is just reflecting the name of the library, I mean the "top-level directory" of the python package).

Reply:
yes. Let's call this "spark_k8s_toolkit"

Reply:
I'm good to do it here. It seems simple/straightforward enough. However I'd leave it to the end of the PR (i.e. when the rest of the requests/code_changes) are all done. (As this one may get annoying on the level of my commits - when combined with additional changes)